export_locally.py

```
@@ -0,0 +1,28 @@
+    )
+
+for quantization_config in ['arm64', 'avx2', 'avx512', 'avx512_vnni']:
+    export_dynamic_quantized_onnx_model(
+        onnx_model,
+        quantization_config=quantization_config,
+        model_name_or_path=output_dir,
+    )
+
+openvino_model = SentenceTransformer(model_id, backend="openvino")
+openvino_model.save_pretrained(output_dir)
+export_to_hub.py
```

Ruff (0.14.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)